neighborhood/views.py

```diff
@@ -1,23 +1,20 @@
 from django.shortcuts import render
-
 from .models import Neighborhood
 from .utils import get_title
-
 from decouple import config
+from django.core import serializers
 
 
 def neighborhoods(request):
     neighborhoods = Neighborhood.objects.all()
-
+    neighborhoods_json = serializers.serialize("json", neighborhoods)
     context = {
-        "neighborhoods": neighborhoods,
         "page": "neighborhoods",
         "MAPBOX_API_KEY": config("MAPBOX_API_KEY"),
     }
-
     if request.user.is_authenticated:
         context["firstname"] = request.user.first_name
-
+    context["neighborhoods"] = neighborhoods_json
```

Comment:
I would suggest to revert this change as it doesn't really improve anything. The previous change just works the same way.

Please let me know if I am missing anything.